README.md

```diff
@@ -45,6 +45,15 @@ for the plugin:
   Use `newPenTemplates` to optionally register additional
   template shortcuts for generating new CodePens on the fly.
 
+JavaScript for interactivity and loading the baseline `support` content are
+included in the default bundle. Make sure it's included in a template. 
+
+```js
+<script @raw="getBundle('js')"></script>
```

Comment:
This is also using webc syntax. Is there a place where this option works?

Reply:
Nope, there isn't. I realized documenting all the ways of doing this was overkill, and just kept the web:c version.